src/app-gocardless/banks/nationwide-naiagb21.js

```diff
@@ -23,8 +23,6 @@ export default {
       transaction.bookingDate = useDate.toISOString().slice(0, 10);
     }
 
-    console.log(transaction);
-
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏thanks!